package/MDAnalysis/topology/PDBParser.py

```diff
@@ -302,15 +304,21 @@ def _parseatoms(self):
                 occupancies.append(float_or_default(line[54:60], 0.0))
                 tempfactors.append(float_or_default(line[60:66], 1.0))  # AKA bfactor
 
-                segids.append(line[66:76].strip())
+                segids.append(line[72:76].strip())
 
         # Warn about wrapped serials
         if self._wrapped_serials:
             warnings.warn("Serial numbers went over 100,000.  "
                           "Higher serials have been guessed")
 
+        # If segids is not equal to chainids, warn the user
+        if any([a != b for a, b in zip(segids, chainids)]):
+            warnings.warn("Segment IDs and Chain IDs are not completely equal.")
+
```

Comment:
Generally, SEGIDs will be 4 characters and ChainIDs will be 1 character. If someone had both in the same file, I assume they wanted them to be different, so it does not make sense to warn about it. So this warning does not make sense to me as it warns when something normal happens.

I am in favor of removing it.

If you or other reviewers have a strong reason to keep it please say so.

Reply:
You could use the logger to log it at INFO or DEBUG.

Reply:
I use a logger to log it at DEBUG.